src/molecule/__init__.py

```diff
@@ -25,8 +25,8 @@
 __metaclass__ = type
 
 try:
-    import pkg_resources
+    import importlib_metadata
 
-    __version__ = pkg_resources.get_distribution("molecule").version
+    __version__ = importlib_metadata.version("molecule")  # type: ignore
```

Comment:
Plz use specific ignores rather than ignoring everything.
```suggestion
    __version__ = importlib_metadata.version("molecule")  # type: ignore[a-specific-problem-id]
```